domenetjenester/arbeidsforhold/src/main/java/no/nav/foreldrepenger/domene/arbeidInntektsmelding/ArbeidsforholdInntektsmeldingMangelTjeneste.java

```diff
@@ -118,6 +120,7 @@ public void lagreManuelleArbeidsforhold(BehandlingReferanse behandlingReferanse,
 
     private boolean finnesAvklaringSomGjelderMangel(ArbeidsforholdMangel mangel, List<ArbeidsforholdValg> alleAvklaringer) {
         return alleAvklaringer.stream()
+            .filter(ak -> ArbeidsforholdKomplettVurderingType.FORTSETT_UTEN_INNTEKTSMELDING.equals(ak.getVurdering()))
```

Comment:
så du ønsker å finne ut om det finnes avklaringer for en mangel, men bare dersom avklaringen er av typen fortsett uten inntektsmelding?

Reply:
Ja, hvis avkalringen er Sett på vent ønsker vi å reutlede aksjonspunkt